### ISSUE TITLE
SC 3.2.6: Consistent Help - Normative area - "If a web page contains" leaves out many of the cases encountered

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/wcag#3683]
The normative text of the SC 3.2.6: Consistent Help starts with 

> If a [Web page](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help#dfn-web-page) contains any of the following help [mechanisms](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help#dfn-mechanism), and those mechanisms are repeated on multiple Web pages within a [set of Web pages](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help#dfn-set-of-web-pages)...

In simpler terms, if a web page lacks these help mechanisms, SC 3.2.6 does not apply.

Now, consider the following scenario: we have a website with a core flow consisting of several steps:
- step 1: homepage - help mechanism available
- step 2: categories page - help mechanism **NOT** available
- step 3: category landing page - help mechanism available
- step 4: products listing page - help mechanism **NOT** available
- step 5: product details page - help mechanism available

According to the normative text, if the help mechanism consistently appears in the same position within pages where it is available, SC 3.2.6 is considered compliant. However, it's evident that in this scenario, adherence to SC 3.2.6 does not ensure a satisfactory user experience; this is likely comparable to a set of pages where the help mechanism is present in varying relative orders.

Perhaps it would be more appropriate to align with SC 3.2.3 Consistent Navigation

> Help [mechanisms] that are repeated on multiple [Web pages](https://www.w3.org/TR/WCAG22/#dfn-web-page-s) within a [set of Web pages](https://www.w3.org/TR/WCAG22/#dfn-set-of-web-pages) occur in the same order, unless a change is initiated by the user...

**Comment by mbgower** [comment]
Draft response:
Neither Consistent Navigation nor Consistent Help require that mechanisms are _used_ on every page in the set of pages. Rather, the success criteria provide guidance for mechanisms' position and order "each time they are repeated". As the Understanding document for Consistent Help states,
>The Criterion only requires that when one of the listed forms of help is available across multiple pages that it be in a consistent location.

The failure technique for Consistent Navigation is titled "[F66: Failure of Success Criterion 3.2.3 due to presenting navigation links in a different relative order on different pages](https://www.w3.org/WAI/WCAG22/Techniques/failures/F66). It does not discussing missing links; just inconsistently ordered links.

It would _seem_ to be a much better experience if, in your scenario, each page in the set of pages included the same help links. However, considering every scenario in which help mechanisms appear on sets of pages, there may be situations where it does not make sense to include the same help or navigation mechanisms on every page in a set of pages. If there is not a good design reason in your case for the help mechanisms to be absent from 2 of the steps, it would create a more consistent,  accessible experience if the mechanisms appeared on all pages.

**Comment by bruce-usab** [comment]
Discussed on Friday call.

**Comment by mbgower** [comment]
Action: we will add an issue to put more context in the  Understanding document and raise as an important consideration to incorporate in WCAG 3
